source/stylesheets/pages/meetup-assets.css.scss

```diff
@@ -1,3 +1,13 @@
+.meetup-assets {
+  .example-meetups-logo {
+    float: right;
+    @include margin(null null 15px 15px);
```

Comment:
@jeffdaley If you run `bundle exec middleman build` locally you'll get:
 
```
bundler: failed to load command: middleman (/usr/local/bin/middleman)
Sass::SyntaxError: Undefined mixin 'margin'.
```

Do we need a `@mixin margin`?

Reply:
I thought that would work with the Bourbon gem. Guess not!

I rewrote in plain css and resubmitted.